QEfficient/base/modeling_qeff.py

```diff
@@ -312,3 +314,99 @@ def _compile(
 
         self.qpc_path = qpc_path
         return qpc_path
+
+    def _qnn_compile(
+        self,
+        onnx_path: Optional[str] = None,
+        compile_dir: Optional[str] = None,
+        *,
+        specializations: Optional[List[Dict[str, int]]] = None,
+        prefill_seq_len: int = 32,
+        ctx_len: int = 128,
+        batch_size: int = 1,
+        full_batch_size: Optional[int] = None,
+        mdp_ts_num_devices: int = 1,
+        num_cores: int = 16,
+        mxfp6_matmul: bool = False,
+        mxint8_kv_cache: bool = False,
+        qnn_config: Optional[str] = None,
+    ) -> str:
+        """
+        Interface for QNN compiler
+
+        Args:
```

Comment:
Can we divide this into mandatory and optional Args for better understanding?

Reply:
Sure, Done.